src/AppBundle/EventListener/PostFacebookAuthenticationListener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace AppBundle\EventListener;
+
+use AppBundle\Event\FacebookLoginEvent;
+use Misteio\CloudinaryBundle\Wrapper\CloudinaryWrapper;
+
+class PostFacebookAuthenticationListener
+{
+    /**
+     * @var CloudinaryWrapper
+     */
+    private $cloudinary;
+
+    /**
+     * PostFacebookAuthenticationListener constructor.
+     * @param CloudinaryWrapper $cloudinary
+     */
+    public function __construct(CloudinaryWrapper $cloudinary)
```

Comment:
Cia siaip toks pastebejimas ateiciai, kad labai decouplinta gaunasi. Gerai butu pasikurti koki savo interface ir naudoti ta interface, kad jeigu reiktu keisti ateityje is cloudinary i koki kita, pvz. amazon uploadinima, tai uztektu vienoj vietoj tiktai keisti, kad visoj sistemoj pasikeistu.